doc/alertmanager-mixin/Makefile

```diff
@@ -1,10 +1,13 @@
 JSONNET_FMT := jsonnetfmt -n 2 --max-blank-lines 2 --string-style s --comment-style s
 ALERTMANAGER_ALERTS := alertmanager_alerts.yaml
 
-default: build
+default: vendor build
 
 all: fmt build
```

Comment:
It would be nice if dashboards were produced as a result of `make` (without specifying any target).